### File look OK
### "both-case-p" are not done, but should be identical with this
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:lower-case-p.txt	2017-04-27 22:03:56.000000000 +0100
+++ saved-pages-clus/cl:functions:lower-case-p.txt	2017-04-27 22:03:56.000000000 +0100
@@ -2,11 +4,11 @@
 Syntax


-upper-case-p character => generalized-boolean
+ upper-case-p character → generalized-boolean
+ lower-case-p character → generalized-boolean
+ both-case-p character → generalized-boolean

-lower-case-p character => generalized-boolean

-both-case-p character => generalized-boolean

 Arguments and Values

@@ -21,20 +39,75 @@
 Examples


- (upper-case-p #\A) =>  true
- (upper-case-p #\a) =>  false
- (both-case-p #\a) =>  true
- (both-case-p #\5) =>  false
- (lower-case-p #\5) =>  false
- (upper-case-p #\5) =>  false
- ;; This next example presupposes an implementation
- ;; in which #\Bell is an implementation-defined character.
- (lower-case-p #\Bell) =>  false

-<TT>
-Side Effects None.
+(upper-case-p #\A)
+→
+true
+(upper-case-p #\a)
+→
+false
+(both-case-p #\a)
+→
+true
+(both-case-p #\5)
+→
+false
+(lower-case-p #\5)
+→
+false
+(upper-case-p #\5)
+→
+false
+;; This next example presupposes an implementation in which #\Bell is an implementation-defined character.
+(lower-case-p #\Bell)
+→
+false
+Side Effects
+None.
+Affected By
+None.

-Affected By None.

 Exceptional Situations

@@ -38,8 +111,9 @@

 Exceptional Situations

 Should signal an error of type type-error if character is not a character.
 See Also

-char-upcase, char-downcase, Section 13.1.4.3 (Characters With Case), Section 13.1.10 (Documentation of Implementation-Defined Scripts)
### char-upcase, char-downcase, Section 13.1.4.3 (Characters With Case) {\secref\CharactersWithCase}
### Section 13.1.10 (Documentation of Implementation-Defined Scripts) {\secref\ImplementationDefinedScripts}
-Notes None.
\ No newline at end of file
+Function CHAR-UPCASE, Function CHAR-DOWNCASE, {\secref\CharactersWithCase}, {\secref\ImplementationDefinedScripts}
+Notes
+None.
\ No newline at end of file
